scripts/build_pkgdown.R

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env Rscript
-
 # Build pkgdown documentation for PEcAn packages
 library(pkgdown)
-
+library(yaml)
 args <- commandArgs(trailingOnly = TRUE)
 if (length(args) == 0) {
   stop("No package names provided. Please pass package names as arguments.")
 }
-
 packages <- args
-
+output_dir <- "_pkgdown_docs"
```

Comment:
If adopting this name here, need to adjust the GHA and Docker configs above to match it